Sources/NIOTransportServices/NIOTSSingletons.swift

```diff
@@ -0,0 +1,79 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2023 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+#if canImport(Network)
+import NIOCore
+
+@available(OSX 10.14, iOS 12.0, tvOS 12.0, watchOS 6.0, *)
+extension NIOTSEventLoopGroup {
```

Comment:
Could you add an extension to `EventLoopGroup` like https://github.com/weissi/swift-nio/blob/2daebdcff77abba902cfdc33eac4e058fe8f3ae8/Sources/NIOPosix/PosixSingletons.swift#L57-L64?

Reply:
done!